test/skimNtuple_HHbtag.cpp

```diff
@@ -1451,8 +1594,12 @@ int main (int argc, char** argv)
 		  int dauType = theBigTree.particleType->at(idau);
 		  if (oph.isMuon(dauType))
 			{
-			  bool passMu   = oph.muBaseline (&theBigTree, idau, 20., 2.4, 0.15, OfflineProducerHelper::MuTight, string("All") , (DEBUG ? true : false));
-			  bool passMu10 = oph.muBaseline (&theBigTree, idau, 10., 2.4, 0.30, OfflineProducerHelper::MuTight, string("All") , (DEBUG ? true : false));
+			  bool passMu   = oph.muBaseline (&theBigTree, idau, 20., 2.4,
+											  0.15, OfflineProducerHelper::MuTight,
+											  0.15, OfflineProducerHelper::MuHighPt, string("All"), (DEBUG ? true : false));
+			  bool passMu10 = oph.muBaseline (&theBigTree, idau, 10., 2.4,
+											  0.30, OfflineProducerHelper::MuTight,
+											  0.30, OfflineProducerHelper::MuHighPt, string("All") , (DEBUG ? true : false));
```

Comment:
it may be worth checking how this does affect the distributions by itself (== outside this PR)